src/app/phase-bug-reporting/new-issue/new-issue.component.html

```diff
@@ -23,15 +23,15 @@ <h1 class="mat-display-1 title">New Issue</h1>
           </app-comment-editor>
         </div>
 
-        <button style="float: right" type="submit" mat-stroked-button color="primary"
+        <button style="float: right" class="submit-new-bug-report" type="submit" mat-stroked-button color="primary"
           [disabled]="!newIssueForm.valid || isFormPending">
           {{ this.submitButtonText }}
         </button>
       </div>
 
       <div class="column right">
-        <app-label-dropdown initialValue="" attributeName="severity" [dropdownForm]="newIssueForm"></app-label-dropdown>
-        <app-label-dropdown initialValue="" attributeName="type" [dropdownForm]="newIssueForm"></app-label-dropdown>
+        <div class="severity-dropdown"><app-label-dropdown initialValue="" attributeName="severity" [dropdownForm]="newIssueForm"></app-label-dropdown></div>
+        <div class="bug-dropdown"><app-label-dropdown initialValue="" attributeName="type" [dropdownForm]="newIssueForm"></app-label-dropdown></div>
```

Comment:
This change is necessary due to an issue with Geckodriver + Firefox where the `app-label-dropdown` component is "unable to be scrolled to" and thus causing the E2E to fail. Tried a bunch of fixes like changing the current frame, forcefully scrolling to the component (even though it was already on screen), and a bunch of other things but it didn't help. Wrapping the component in its own `div` seemed to be the easiest and only working way I could come up with 😅 There doesn't really seem to be any other working solutions online.